azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -36,6 +36,7 @@
     "IonQQPUQirBackend",
     "IonQQPUNativeBackend",
     "IonQAriaQirBackend",
+    "IonQForteBackend",
```

Comment:
Please see Aria as a reference. There are two backends defined for IonQAriaQirBackend and IonQAriaNativeBackend. Please follow the same template for aria.